migration/migrator/migrations/master/20220811134614_user_session_management.py

```diff
@@ -0,0 +1,35 @@
+"""Migration for the Submitty master database."""
+
+def up(config, database):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    """
+
+    database.execute("""
+        BEGIN;
```

Comment:
You don't need to include `BEGIN` and `COMMIT` in the migration.  The migrator does that for you.  You just need to make sure all of the commands are in a single `database.execute()` call.

Reply:
made the changes :)

Reply:
Thanks for your insight :)